wdio-local.conf.js

```diff
@@ -291,6 +291,13 @@ exports.config = {
             }, this);
         }, true);
 
+        browser.addCommand('addIsActiveClass', function() {
+            browser.execute(function(domElement) {
+                domElement.classList.add('is-active');
+
```

Comment:
Nit: Could you please remove this empty line?

Reply:
done